feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -100,4 +100,8 @@ export class ConfiguratorFormComponent implements OnInit {
   createGroupId(groupId?: string): string | undefined {
     return this.configUtils.createGroupId(groupId);
   }
+
+  getComponentKey(attribute: Configurator.Attribute): string {
+    return 'AttributeType_' + attribute.uiType;
```

Comment:
Nitpick: let's move string `'AttributeType_'` to a separate `protected readonly` property